### DISCUSSION TITLE
ViTPose Model

### DISCUSSION THREAD

**Discussion by gusmcarreira** [open-mmlab/mmpose#2421] [General] [answered]
I'm trying to use this config file, "https://github.com/open-mmlab/mmpose/blob/main/configs/body_2d_keypoint/topdown_heatmap/coco/td-hm_ViTPose-large_8xb64-210e_coco-256x192.py" but in the backbone I don't get the type being "mmcls.VisionTransformer"? Any help, what should I swap it for?

Kind regards

**Comment by LareinaM** [comment, marked as answer]
Hi, we use the `VisionTransformer` implemented by [mmcls (mmpretrain)](https://github.com/open-mmlab/mmpretrain/blob/1.x/mmcls/models/backbones/vision_transformer.py). Please follow their installation guide.

**Reply by seon-creator** [reply]
Hello, I also want to use that model, where is the following the installation guide?
I'm using mmpose but there is mmpretrain.
Just copy code?
Or another installation command?
